src/puppetlabs/puppetdb/http/fact_names.clj

```diff
@@ -25,7 +25,8 @@
                  version
                  (http-q/validate-distinct-options! (merge (keywordize-keys params) puppetdb-query))
                  (:scf-read-db globals)
-                 (:url-prefix globals))))
+                 (:url-prefix globals)
+                 (:pretty-print globals))))
```

Comment:
I don't know I think it's pretty clear that trying to pretty-print a json string will have no effect